index.html

```diff
@@ -0,0 +1,198 @@
+      <p class="item"> 
+        I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+      </p>
+      <div class="socials item">
+        <i class="fab fa-brands fa-twitter"></i>
+        <i class="fab fab-brands fa-linkedin-in"></i>
+        <i class="fab fa-brands fa-github"></i>
+        <i class="fa fa-regular fa-hand-peace"></i>
+      </div>
+      
+      <button class="arrow item"><i class="fa fa-solid fa-chevron-down"></i></button>
+      <h2 class="item">My Recent Works</h2>
+      <hr/>
+    </section>
+    <section class="card grid">
+      <div class="item img-holder grid-item-1">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-2">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-3">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li  class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+          </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-4">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
+          <ul class="row">
+            <li class="but"><span class="tab">Ruby on rails</span></li>
+            <li class="but"><span class="tab">css</span></li>
+            <li class="but"><span class="tab">JavaScript</span></li>
+            <li class="but"><span class="tab">html</span></li>
+            </ul>
+          <button class="see">See Project</button>
+        </div>
+      </div>
+
+      <div class="item img-holder grid-item-5">
+        <div class="text">
+          <hx class="topic">Multi-Post Stories Gain+Glory</hx>
```

Comment:
- Kindly, when the previous code reviewer suggested using `hx` means `h1`-`h6` :+1: :+1:  update it because `hx` unkown tag